perfmetrics/scripts/testing_on_gke/examples/dlio/run_tests.py

```diff
@@ -15,34 +15,68 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""This program takes in a json dlio test-config file and generates and deploys helm charts."""
+
+import argparse
 import subprocess
+import dlio_workload
+
 
 def run_command(command: str):
-    result = subprocess.run(command.split(" "), capture_output=True, text=True)
-    print(result.stdout)
-    print(result.stderr)
-
-metadataCacheTtlSecs = 6048000
-bucketName_numFilesTrain_recordLength_batchSize = [
```

Comment:
Replacement for this hardcoded set of tests is in https://github.com/GoogleCloudPlatform/gcsfuse/pull/2264/files .